packages/vaadin-grid-pro/test/typings/grid-pro.types.ts

```diff
@@ -1,59 +1,38 @@
-import '../../src/vaadin-grid-pro';
-import '../../src/vaadin-grid-pro-edit-column';
-import { GridColumnElement, GridDropLocation, GridItem, GridItemModel } from '@vaadin/vaadin-grid';
+import { GridColumnElement, GridElement } from '@vaadin/vaadin-grid';
+import { GridProElement } from '../../src/vaadin-grid-pro';
+import { GridProEditColumnElement } from '../../src/vaadin-grid-pro-edit-column';
```

Comment:
Let's change these to use root level entrypoints as descibed in #181 to ensure public exports work.

```suggestion
import { GridProElement } from '../../vaadin-grid-pro';
import { GridProEditColumnElement } from '../../vaadin-grid-pro-edit-column';
```

Reply:
OK, changed this for all elements in type tests.

One other issue that I found was the `src/interfaces.d.ts` re-exported `GridColumnElement`, which seemed wrong as it was the only element that was re-exported. I removed the export and changed the test file `grid-pro.types.ts` to import the type from `@vaadin/vaadin-grid/vaadin-grid-column`. But I don't know if this will have other side-effects.